projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/commandsAndMenu.tsx

```diff
@@ -200,4 +204,74 @@ export function updateVDKMenu(commands: CommandRegistry) {
       }
     }
   });
+
+  commands.addCommand('jp-vdk:menu-create-deployment', {
+    label: 'Deploy',
+    caption: 'Create deployment of a VDK job',
+    execute: async () => {
+      try {
+        if (!runningVdkOperation) {
+          runningVdkOperation = true;
+        const result = await showDialog({
+          title: 'Create Deployment',
+          body: (
+            <DeployJobDialog jobName='job-to-deploy' jobPath={sessionStorage.getItem('current-path')!} jobTeam='my-team' vdkVersion='0.11'></DeployJobDialog>
```

Comment:
what's the plan for populating the VDK version?

Reply:
Mmm, vdk version should generally be left by default. 

VDK version option during deploy is used if for some reason (usually due to major version upgrade of vdk user wants to use older version of vdk temporarily) but in most cases you want the version controlled by the platform team. 

The vdk version should be hidden from the UI, and is definitely adding it in the UI is very low priority. 

It must not be inferred from the environment.

Reply:
I agree that the version should be initially hidden. After the user clicks show/advanced in the "Create data job" form, then the field should be "empty" (default). In case empty, the version should not be sent to the control service;


However showing/hiding optional fields, or having advanced configurations -> does not sound urgent to me. It would also require an (i) icon, for explaining what the purpose is.  The (i) icon should emphasise that the data job may be tested locally with a particular version -and running in the cloud with the latest. Maybe a warning, to prevent discrepancies.
Like a user, I would expect to be testing the workload with the same VDK version that runs in the cloud.

Reply:
I will remove it for now, because of the low priority. I will create an issue for it. Since showing/hiding options might take time and the purpose of this PR is to introduce the main functionality for Deployment.

Reply:
![Screenshot 2023-03-07 at 14 09 06](https://user-images.githubusercontent.com/87015481/223418031-8a16e727-99db-455a-9d35-793559876c9a.png)
It is like that rn

Reply:
nit:  the job be enabled by default